frontend/javascripts/oxalis/view/jobs/train_ai_model.tsx

```diff
@@ -250,8 +251,8 @@ export function TrainAiModelTab<GenericAnnotation extends APIAnnotation | Hybrid
       await ensureSavedState();
     }
 
-    await runTraining({
-      trainingAnnotations: await getTrainingAnnotations(values),
```

Comment:
was this on purpose? you re-added the first await again [here](https://github.com/scalableminds/webknossos/pull/8266/commits/63258289b8889f5628300b93c5a3c93d1bd43a0f), but the second one not? I think, both awaits are important?